sceptre/cli/template.py

```diff
@@ -137,3 +138,33 @@ def fetch_remote_template_command(ctx, path):
             output.append(template)
 
     write(output, context.output_format)
+
+
+@click.command(name="detect-stack-drift", short_help="Detects stack drift on running stacks.")
+@click.argument("path")
+@click.pass_context
+@catch_exceptions
+def detect_stack_drift_command(ctx, path):
+    """
+    Detect stack drift on running stacks.
```

Comment:
I think a most extensive docstring is important here. This doesn't just _detect_ drift; It shows resource drifts.